posthog/warehouse/models/external_data_schema.py

```diff
@@ -218,6 +218,14 @@ def soft_delete(self):
         self.deleted_at = datetime.now()
         self.save()
 
+    def delete_table(self):
```

Comment:
I presume this won't actually delete any data in S3 then?

Reply:
and could this cause issues with consistency if they then resync this schema in the future (eg old data appearing)?